backend/data/src/main/kotlin/io/tolgee/repository/UserAccountRepository.kt

```diff
@@ -13,6 +14,85 @@ import org.springframework.data.jpa.repository.Query
 import org.springframework.stereotype.Repository
 import java.util.*
 
+const val USER_FILTERS = """
```

Comment:
Why on the bottom of the file? For me it make sense to declare something before I use, otherwise it feels like I'm using something that doesn't exist yet (just a code style feeling).